CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/generic-msp430-gcc.cmake")
+
+# Name your project
+project(EXAMPLE)
+
+#device dependent config, here for launchpad
+set(MSP430_MCU msp430fr2355)
+set(MSP430_MCU_FREQ 24000000UL)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+# set(CMAKE_C_EXTENSIONS ON)
+
+add_compile_options(-Werror -ffunction-sections -fdata-sections -mmcu=${MSP430_MCU} -DF_CPU=${MSP430_MCU_FREQ})
+
+
+# note-c
+set(NOTE_C ${CMAKE_CURRENT_LIST_DIR}/note-c)
+add_library(note-c OBJECT
+    ${NOTE_C}/n_atof.c
+    ${NOTE_C}/n_b64.c
+    ${NOTE_C}/n_cjson.c
+    ${NOTE_C}/n_cjson_helpers.c
+    ${NOTE_C}/n_const.c
+    ${NOTE_C}/n_ftoa.c
+    ${NOTE_C}/n_helpers.c
+    ${NOTE_C}/n_hooks.c
+    ${NOTE_C}/n_i2c.c
+#    ${NOTE_C}/n_md5.c
```

Comment:
May be useful to add a comment explaining why n_md5.c and n_ua.c are commented out.

Reply:
will do. note-c hasn't been updated in this project for some time. I'll probably do that in a separate PR.